modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AspNetServerCodegen.java

```diff
@@ -783,6 +793,14 @@ private void setIsFramework() {
             }
             additionalProperties.put(TARGET_FRAMEWORK, "netcoreapp" + aspnetCoreVersion);
         }
+
+        setAddititonalPropertyForFramework();
+    }
+
+    private void setAddititonalPropertyForFramework() {
+        if (((String)additionalProperties.get(TARGET_FRAMEWORK)).startsWith("net6.0")) {
```

Comment:
startsWith("net6.0") fails to capture the or later part of NET_60_OR_LATER. Also, I think the target framework could contain multiple frameworks. 

This is something that really should be in the abstract and not needed here, but sadly it is not :(

Reply:
-> startsWith("net6.0") fails to capture the or later part of NET_60_OR_LATER
Do you mean versions like net7.0, net.8.0? aspnetcore generator supports only 6.0 as for now.
-> Also, I think the target framework could contain multiple frameworks.
`targetFramework` from csharp generator could, indeed. But aspnetcore generator use `aspnetCoreVersion` parameter, which couldn't.
-> This is something that really should be in the abstract and not needed here, but sadly it is not :(
This is because of different parameters and logic around it. Also, csharp generator supports .netstandard and .NET framework, but aspnetcore doesn't. So merge it in abstract class will break LS solid principe.

Reply:
I would still suggest handling past .net6. The end of life on it is this year. Surprising to me that no one has updated it already. Other than that, LGTM.

Reply:
Done.
-> Surprising to me that no one has updated it already
I think it's because aspnetcore generator is not popular since it's not what you can generate on every spec update.